ReleaseProcess.md

```diff
@@ -27,7 +33,9 @@ Image automatically synched to MCR CN from Public cloud MCR.
 
 ## AKS
 
-Make PR against [AKS-RP](https://msazure.visualstudio.com/CloudNativeCompute/_git/aks-rp?version=GBmaster) repo with chart update(s)
+- Make PR to prebake the latest image (https://github.com/Azure/AgentBaker)
+- Make PR against [AKS-RP](https://msazure.visualstudio.com/CloudNativeCompute/_git/aks-rp?version=GBmaster) repo with chart update(s)
```

Comment:
This is public facing. We should probably not doc how we deploy to AKS here.

Reply:
> This is public facing. We should probably not doc how we deploy to AKS here.

Make sense.  Let me remove these and we can add it to our internal wiki

Reply:
> > This is public facing. We should probably not doc how we deploy to AKS here.
> 
> Make sense. Let me remove these and we can add it to our internal wiki

> removed.